packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1829,6 +1859,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
      * @param options - options controlling how the summary is generated or submitted
      */
     public async submitSummary(options: ISubmitSummaryOptions): Promise<SubmitSummaryResult> {
+        // increment summary count
+        this.summaryCount++;
```

Comment:
This should probably be called `submittedSummaryCount`. Also, I am thinking if we should only increment it on successfully submitting a summary.
@vladsud We are trying to get the number of summaries submitted to the server right? This function can return without submitting the summary. So, this should be incremented **_after_** the summary is submitted.

Reply:
I am wondering if we should consider the scenario where we summarize via some other route. In that case, should we skip adding this count to the metadata blob. That would avoid having to update the test snapshots [here](https://github.com/microsoft/FluidFrameworkTestData/pull/28).